numpy/_core/defchararray.pyi

```diff
@@ -566,12 +599,19 @@ def replace(
     count: None | i_co = ...,
 ) -> NDArray[str_]: ...
 @overload
+def replace(
+    a: B_co,
+    old: B_co,
+    new: B_co,
+    count: None | i_co = ...,
+) -> NDArray[bytes_]: ...
```

Comment:
It's probably best for consistency to leave it like it is, but in case you didn't know already: 
The official typing docs recommend using a `130` character limit in `.pyi`, so I'm guessing that this could fit on one line.

https://typing.readthedocs.io/en/latest/guides/writing_stubs.html#maximum-line-length